paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerConfigurationHandler.kt

```diff
@@ -27,6 +27,14 @@ internal class FlowControllerConfigurationHandler @Inject constructor(
 
     private val job: AtomicReference<Job?> = AtomicReference(null)
 
+    private var didLastConfigurationFail: Boolean = false
+
+    val isConfigured: Boolean
```

Comment:
Future me thinks this name is a little bit misleading, since when I'm reading code in the FlowController. I wouldn't think this would be true when `isConfiguring`.

I don't have a great name replacement, but I was thinking it could be named `isInActionableState`, `canPerformAction`, etc.